{{cookiecutter.project_name}}/makefile

```diff
@@ -0,0 +1,8 @@
+.PHONY: all
+all:
+	$(info Generating the JDF PDF.)
+	pdflatex main
```

Comment:
Shouldn't this be `main.tex`?

Likewise in all other commands?

Reply:
It auto detects that.

Reply:
Sounds good. Thanks!